packages/odf/components/kms-config/kms-config.tsx

```diff
@@ -34,7 +34,7 @@ const KMSProviders: KMSProvidersType = [
     Component: ThalesConfigure,
   },
   {
-    name: 'Azure Key Vault',
+    name: 'Azure Key Vault (Tech preview)',
```

Comment:
Do we need translations for this, right?
```suggestion
    name: t('Azure Key Vault (Tech preview)'),
```